share/p2p/shrexnd/params.go

```diff
@@ -8,7 +8,7 @@ import (
 	"github.com/celestiaorg/celestia-node/share/p2p"
 )
 
-const protocolString = "/shrex/nd/v0.0.2"
```

Comment:
It would be nice to merge this PR plus the #2444 together so that we can break shrexnd one time.